test_discovery.py

```diff
@@ -272,18 +282,21 @@ def validate_entities():
         except FileNotFoundError:
             pass
 
+    # remove mocked up dummy module from modules
+    sys.modules.pop('nlp.cleanText')
+
 
 def _validate_individual_entity(entity):
-  entity_name = os.path.split(entity)[-1].replace(".py", "")
-  entity_module = import_module(entity_name)
-  definition_errors = []
-  if 'ENTITY_DEFINITION' in dir(entity_module):
-      print('Checking entity definition {0:.<35}'.format(entity_name), end='')
-      errors = find_errors_in_entity_definition(entity_module.ENTITY_DEFINITION)
-      _log_entity_definition_error_results(errors)
-      definition_errors.extend(errors)
-  if definition_errors:
-    raise Exception('Please fix all entity definition errors before running discovery!')
+    entity_name = os.path.split(entity)[-1].replace(".py", "")
```

Comment:
Are these extra spaces intentional?

Reply:
We use 4 spaces on the open-source stuff to conform to PEP styling.

Reply:
Great!